v2/custom-elements/android-breakage-dialog.js

```diff
@@ -0,0 +1,167 @@
+import { css, html, LitElement } from 'lit'
+import { map } from 'lit/directives/map.js'
+import '@material/web/button/filled-button.js'
+import '@material/web/button/filled-tonal-button.js'
+import '@material/web/button/text-button.js'
+import '@material/web/dialog/dialog.js'
+import '@material/web/icon/icon.js'
+import '@material/web/iconbutton/icon-button.js'
+import '@material/web/radio/radio.js'
+import '@material/web/textfield/outlined-text-field.js'
+
+export class AndroidBreakageDialog extends LitElement {
+    static properties = {
+        items: {
+            type: Array,
+        },
+        title: {
+            type: String,
+        },
+        opened: {
+            type: Boolean,
+        },
+    }
+
+    title = 'please set the title prop'
+    /** @type {[string, string][]} */
+    items = []
+    opened = false
+
+    /**
+     * Every time we show the modal, reset the form to
+     * ensure it's in sync with the one in the page
+     *
+     * @param {string} initial - the category value
+     */
+    show(initial) {
+        if (!this.shadowRoot) throw new Error('unreachable')
+        const elem = this.shadowRoot?.querySelector('md-dialog')
+        if (!elem) throw new Error('unreachable')
+
+        const form = elem?.querySelector('form')
+        if (!form) throw new Error('unreachable, missing form')
+
+        const radioElements = this.shadowRoot?.querySelectorAll('md-radio')
+
+        form.reset()
+
+        for (let element of Array.from(radioElements)) {
+            if (element.value === initial) {
+                element.checked = true
+            }
+        }
```

Comment:
this is needed because the material dialog stays around when closed, so the state can become out of sync with the page below